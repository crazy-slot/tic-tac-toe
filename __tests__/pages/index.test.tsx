import Index from '~pages/index';
import testProvider from '~utils/TestHelper';

describe('Testing pages Index', () => {
	it('should print Index page', () => {
		const { container } = testProvider(<Index />);

		expect(container).toMatchSnapshot();
	});
});
