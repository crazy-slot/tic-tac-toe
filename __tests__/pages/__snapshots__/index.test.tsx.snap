// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing pages Index should print Index page 1`] = `
<div>
  <main>
    tic-tac-toe
  </main>
</div>
`;
